src/components/App.js

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
-
+import React, { Component } from 'react';
 import Router from './Router';
 
-const App = () => <Router />;
+import { catchErrorsWithScope } from 'utils/sentryDSN';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError() {
+    return { hasError: true };
+  }
+
+  componentDidCatch(error, errorInfo) {
+    catchErrorsWithScope(error, errorInfo);
+  }
+
+  render() {
+    if (this.state.hasError) {
+      return (
+        <div>
```

Comment:
Let's create a separate component for the error page.
It should go like components/common/errorPage.js

Reply:
Also, let's keep the `App.js` file clean and make another component for the ErrorBoundary. Add `componentDidCatch` and the `hasError` state to the`ErrorBoundary` component.

Create an `errors` folder in `common`; `components/common/errors` and keep these components there.

```js
 // App.js
render() {
  return (
    <ErrorBoundary>
      <Router/>
     <ErrorBoundary/>
  )
}
```

Reply:
Done!